Makefile

```diff
@@ -72,6 +72,7 @@ log:
 
 dl-models:
 	@echo "🥫 Downloading models …"
+	chmod a+w models
```

Comment:
I added this just to get my `make` command when first checking out the code to work. Has anyone else experienced that problem? This doesn't feel like the correct solution.

Reply:
Should I have done that manually? Is there no way to do it automatically?

Reply:
Super annoying, there should always be a default that just works I feel. But I'll remove the chmod :)

Reply:
I don't think we ***need*** to set `OFF_UID` for Robotoff --> this adds a step that is slowing local dev imo. For PO we do need to change ownership of data folders coming from external, but for Robotoff I don't think it should be necessary.

Reply:
I disagree, in the sense, that you will end up with folders on your machine, that you can't list / copy etc. without using sudo, which seems worst to me. (but yes it's a compromise to be found)

For me as soon as you:
- don't use root in your container (security best practice)
- are a local dev (not on machine, where you handle multi user and usually connect with root / sudoer) (this may apply if you admin stuff with non sudoers users on the system)
- bind mount folders in docker

You have to align container user and your id.

Typically here, for the models folders, how do you handle it otherwise, appart from setting `chmod a+x models/` or `sudo chown 1000:1000 models/` ?

Reply:
Git only seems to care about executable permissions unfortunately.

Do we even _need_ OFF_UID for Robotoff? Why not just run everything as current user?